charts/README.md

```diff
@@ -1,47 +0,0 @@
-# Stream-Registry Helm Chart
-This helm chart creates a [ExpediaGroup Stream-Registry server](https://expediagroup.github.io/stream-registry/).
```

Comment:
Do we not want to offer helm chart in the new offering OOTB? (out of the box) ?
Ok, if we move this to a backlog... (e.g. create an issue that records this and have commitment that we will do this soon).  

THESIS: Having a helm chart helps with adoption and developers "kicking the tires".

Reply:
Absolutely. Yes we should create an issue to ensure it gets readded.

Reply:
Perfect.